apps/suite-launch/test-presets/golden-path-v01-11-2023/mountebank.json

```diff
@@ -0,0 +1,121 @@
+{
+    "port": 4545,
+    "protocol": "https",
+    "stubs": [
+      {
+        "predicates": [{
+          "and": [
+            {
+              "equals": {
+                "path": "/",
+                "method": "POST",
+                "headers": {
+                  "Content-Type": "application/x-amz-json-1.1",
+                  "X-Amz-Target": "Logs_20140328.PutLogEvents"
+                }
+              }
+            }
+          ]
+        }],
+        "responses": [
+          {
+            "is": {
+              "statusCode": 200,
+              "headers": {
+                "x-amzn-RequestId": "d35a6920-52e1-4f3a-bf8f-46016f474463",
+                "Connection": "keep-alive",
+                "Keep-Alive": "timeout=5",
+                "Content-Type": "application/x-amz-json-1.1",
+                "Content-Length": "80"
+              },
+              "body": "{\"nextSequenceToken\":\"49637338213634440402126044667090232594261646769605574882\"}",
+              "repeat": 5
+            }
+          }
+        ]
+      },
+      {
+        "responses": [
+          {
+            "is": {
+              "statusCode": 200,
+              "headers": {
+                "x-amzn-RequestId": "d35a6920-52e1-4f3a-bf8f-46016f474463",
+                "Connection": "keep-alive",
+                "Content-Type": "application/x-amz-json-1.1",
+                "Content-Length": 0,
+                "Date": "Mon, 13 Mar 2023 23:09:24 GMT",
+                "Keep-Alive": "timeout=5"
```

Comment:
is this the value a real CW connection would set?

Reply:
Yes. I tested cloudwatch api on Postman, and it returns timeout = 5